src/main/scala/edu/cmu/cs/obsidian/Main.scala

```diff
@@ -221,7 +227,8 @@ object Main {
                 "rm " + gradleBackupPath.toString
 
             replaceClassNameInGradleBuild.!
-            deleteSedBackupFile.!
+            new File(gradleBackupPath.toString).delete()
+//            deleteSedBackupFile.!
```

Comment:
Delete this whole variable and this line rather than commenting it out and leaving dead code.